src/main/java/org/kainos/ea/health/temp.txt

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
?

Reply:
empty file here to ensure that folder exist